jscomp/main/dune

```diff
@@ -6,7 +6,7 @@
  (modules melc melc_cli)
  (preprocess
   (action
-   (run cppo "%{env:CPPO_FLAGS=}" %{input-file}))))
+   (run cppo "%{env:CPPO_FLAGS=}" -D BS_BROWSER %{input-file}))))
```

Comment:
Are we defining this unconditionally? Doesn't this break our non-browser compiler?

Reply:
No, this was a remaining from previous experiments that I forgot to remove. Did in https://github.com/melange-re/melange/pull/538/commits/df0a373a9204b3b4a47a02e99065f939f8f3ebba.